lib/fix-me.js

```diff
@@ -12,20 +12,24 @@ function FixMe() { }
 FixMe.prototype.runEngine = function(){
   var analysisFiles = [],
       config = {
-        include_paths: ["./"],
+        includePaths: ["./"],
```

Comment:
I would probably leave all the `include_paths` as-is, personally. Loosely speaking they do map to JSON keys:

> (except in cases where it's a key that maps to JSON or something)

Also, I'd probably characterize that as drive-by, whereas `userConfig` is something you're adding now, so it should follow the guide.